.eslintrc.js

```diff
@@ -12,5 +14,9 @@ module.exports = {
     ],
     "no-use-before-define": ["off"],
     "no-useless-constructor": ["off"],
+    // TODO(@decentralion): Enable this rule.
+    //    "flowtype/no-mutable-array": [2],
+    // TODO(@decentralion): Enable this rule.
+    //    "flowtype/require-exact-type": [2, "always"],
```

Comment:
Cool. There are definitely other rules that I would like to add. I’ll PR
them after this is landed.